packages/shared/src/components/cards/PostList.tsx

```diff
@@ -55,11 +58,20 @@ export const PostList = forwardRef(function PostList(
     >
       <CardButton title={post.title} onClick={onPostCardClick} />
       <ListCardMain>
-        <SourceButton
-          source={post?.source}
-          className="mb-2.5"
-          tooltipPosition="top"
-        />
+        {isCollectionPost && (
```

Comment:
We could probably do with ternary here?

Reply:
I prefer this option, ternaries get messy fast 😄